core/app/[locale]/(default)/account/(tabs)/orders/page.tsx

```diff
@@ -0,0 +1,32 @@
+import { notFound } from 'next/navigation';
+
+import { OrdersContent } from './_components/orders-content';
+import { getCustomerOrders } from './page-data';
+
+interface Props {
+  searchParams: Promise<{
+    [key: string]: string | string[] | undefined;
```

Comment:
Is this required? 

Edit: Ohh, it's because we get the order id. I think we should use slugs and individual pages per order, not search params.

So to view an individual order it should be `/account/orders/[id]`